apps/server/src/api-data/sheets/sheets.validation.ts

```diff
@@ -25,7 +24,7 @@ export const validateRequestConnection = [
 ];
 
 export const validateSheetId = [
-  param('sheetId').exists().isString(),
+  param('sheetId').isString(),
```

Comment:
is `exists()` not still required here (and other places?)

Reply:
no, as far as I can understand, the other validators like isString will look make sure the parameter is there